README.Rmd

```diff
@@ -121,12 +121,19 @@ terra::plotRGB(res, 4, 3, 2, scale = max(landsat[["maxValues"]]))
 
 ## Authorization and publication
 
-### Authorization for non-public data access
+Authorization is not required for reading data from any public ArcGIS Online or ArcGIS Enterprise service. 
```

Comment:
```suggestion
Authorization **is not required** for reading data from any public feature services that might be hosted on ArcGIS Online, ArcGIS Hub, ArcGIS Enterprise, or Open Data Portals. 
```
i've bolded the "is not required" section here to get it across. I wonder if it is necessary to list out all of the possible places that data might be accessed? I feel like by mentioning products it starts to feel limiting again. 

@hrecht, do you mind if I ask for your opinion on this? Do you think the current suggested change is sufficient? Or should it be simplified such as 

> Authorization **is not required** for reading any public data sources.

Reply:
I think "Authorization is not required for reading any public data sources." is great! Thanks for tackling this so quickly.

Reply:
Appreciate the feedback - done!